api/src/paths/alert/{alertId}/index.ts

```diff
@@ -0,0 +1,310 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { SYSTEM_ROLE } from '../../../constants/roles';
+import { getDBConnection } from '../../../database/db';
+import { systemAlertSchema } from '../../../openapi/schemas/alert';
+import { authorizeRequestHandler } from '../../../request-handlers/security/authorization';
+import { AlertService } from '../../../services/alert-service';
+import { getLogger } from '../../../utils/logger';
+
+const defaultLog = getLogger('paths/alert/{alertId}/index');
+
+export const GET: Operation = [
+  authorizeRequestHandler(() => {
+    return {
+      and: [
+        {
+          discriminator: 'SystemUser'
```

Comment:
Just checking: 

Do we expect a random user to need to fetch an individual alert? Or should this be restricted to admins?

The only consideration is that the current repo function this calls doesn't block expired alerts. Which is maybe fine? But maybe not something we want random users to be able to fetch?

If Alerts never contain sensitive information, then no issue of course.

Reply:
Not right now - I changed all the alert/{alertId} endpoints to be data/system admins only.